### ISSUE TITLE
Error compiling the project

### ISSUE THREAD

**Issue by callmeahab** [LeonHartley/Coerce-rs#22]
I'm getting an error when trying to compile coerce:

```
error[E0277]: the trait bound `valuable::Value<'_>: tracing::Value` is not satisfied
  --> /github/home/.cargo/registry/src/github.com-1ecc6299db9ec823/coerce-0.8.9/src/actor/lifecycle.rs:84:28
   |
84 |                   let span = tracing::info_span!(
   |  ____________________________^
85 | |                     "actor.recv",
86 | |                     ctx = log.as_value(),
87 | |                     message_type = msg.name(),
88 | |                 );
   | |_________________^ the trait `tracing::Value` is not implemented for `valuable::Value<'_>`
```

And a bunch of other similar issues.

Is this related to `tracing` package version perhaps?

These are relevant packages in `Cargo.toml`:

```toml
tracing = "0.1"
coerce = { version = "0.8.9", features = ["full"] }
coerce-macros = "0.2.0"
```

Thanks

**Comment by LeonHartley** [comment]
Hey @callmeahab. I'll be publishing a release today which should help simplify the process of getting started. It seems `valuable` stabilisation in the `tracing` is a long way off, so I won't be relying on it for now.

Fixed via #23

**Comment by callmeahab** [comment]
Thanks @LeonHartley, I had tracing_unstable cargo flag enabled previously and only realized it failed in docker.
